cobigen-plugins/cobigen-openapiplugin-parent/cobigen-openapiplugin/src/test/java/com/devonfw/cobigen/openapiplugin/integrationtest/InputReaderMatcherTest.java

```diff
@@ -253,17 +254,9 @@ public void testVariableAssignment_noAttributeFound() throws Exception {
 
     File targetFolder = this.tmpFolder.newFolder();
     GenerationReportTo report = cobigen.generate(inputObjects.get(0), template, targetFolder.toPath());
-    assertThat(report).isSuccessful();
-
-    assertThat(targetFolder.toPath().resolve("testVariableAssignment_attribute.txt").toFile()).exists()
-        .hasContent("testingAttributeTable");
-
-    template = findTemplate(cobigen, inputObjects.get(1), templateName);
-    targetFolder = this.tmpFolder.newFolder();
-    report = cobigen.generate(inputObjects.get(1), template, targetFolder.toPath());
-    assertThat(report).isSuccessful();
+    assertThat(report).containsException(CobiGenRuntimeException.class);
 
-    assertThat(targetFolder.toPath().resolve("testVariableAssignment_attribute.txt").toFile()).exists().hasContent("");
+    assertThat(targetFolder.toPath().resolve("testVariableAssignment_attribute.txt").toFile()).doesNotExist();
```

Comment:
Why is the test result changed? It's an existing good case, which should not be a failure after this change, shouldnt it?

Reply:
you are right, the report should just contain warnings.